### PR TITLE
Feature/CORE-2299 metrics fixes

### PR DESCRIPTION
**Pull request by codycoggins-virtru**
* Update metric names and service ID per Dinesh. 
* Update schema/ configuration page
* Handle ubbagent image path more correctly. UbbAgent image has been pushed to our gc.io repo to support this (appears to be current best practice).
This version of the deployer was submitted to Google for review today (3/18/2021)

### CODE REVIEW

File: schema.yaml [hunk 1]
```diff
@@ -53,38 +58,65 @@ properties:
     title: Gateway Hostname
     description: The hostname for your Gateway instance
     type: string
+    title: Gateway Hostname
```
Comment (johndietz) [inline schema.yaml:61]: i think a few of these need deduplication - i see there's already a title/description for some of these adds.

Comment (codycoggins-virtru) [reply schema.yaml:61]: Merge conflicts I missed, thanks!

File: schema.yaml [hunk 2]
```diff
@@ -16,6 +16,11 @@ x-google-marketplace:
           type: REPO_WITH_REGISTRY
         image.tag:
           type: TAG
+    ubbagent:
+      properties:
+        ubbagentImage:
+          type: FULL
+          default: "gcr.io/cloud-marketplace-tools/metering/ubbagent:latest"
```
Comment (ehorning) [inline schema.yaml:23]: can we/should we be using a pinned version?

**Review by johndietz — APPROVED** [review]

File: schema.yaml (hunk 2 above)
Comment (codycoggins-virtru) [reply schema.yaml:23]: Good point. I'm going to make an update on this, there is a marketplace practice I've seen often where we serve the sidecar image from our repo, thus effectively pinning it.

**Review by ehorning — APPROVED** [review]

**Pull request merged**